src/extraction/FStar.Extraction.ML.Syntax.fs

```diff
@@ -224,7 +224,10 @@ let ml_string_ty  = MLTY_Named ([], (["Prims"], "string"))
 let ml_unit    = with_ty ml_unit_ty (MLE_Const MLC_Unit)
 let mlp_lalloc = (["SST"], "lalloc")
 let apply_obj_repr :  mlexpr -> mlty -> mlexpr = fun x t ->
-    let obj_repr = with_ty (MLTY_Fun(t, E_PURE, MLTY_Top)) (MLE_Name(["Obj"], "repr")) in
+    let obj_ns = if Option.get (Options.codegen()) = "FSharp" 
```

Comment:
Should this be Util.codegen_fsharp()?

Reply:
IIRC I was having issues with `Util.codegen_fsharp()`, which is why the body of the function is repeated here. I will look into this in more detail.

Reply:
`Util.codegen_fsharp()` is defined in `FStar.Extraction.ML.Util`, which depends on `FStar.Extraction.ML.Syntax`, so I can't use `Util.codegen_fsharp()` here.

Reply:
Oh, I see. Could we just move it to FStar.Options? Sounds like it should be there anyway, right?

Reply:
That seems like a good idea. There's probably more to be done with regards to handling differences between F# and OCaml when extracting, and `Util.codegen_fsharp()` would be useful for these patches.

Reply:
Done in b35d918